validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/ConfigSequences.java

```diff
@@ -22,6 +27,39 @@
  */
 public class ConfigSequences extends SequenceRunner {
 
+  @Test
+  @Description("Push endpoint config message to device that results in a connection error.")
```

Comment:
these should be in their own class (not in ConfigSequences.java) -- ConfigSequences is scoped at things for basic handling of config messages that don't relate to any specific subblock (e.g. blobs or endpoint). The target class could either be about testing "blobsets" or "endpoint" -- not sure which is more appropriate (since it's really both in this instance, but not always both in the future).